src/MCPServer/lib/workflow_abilities.py

```diff
@@ -0,0 +1,46 @@
+"""The abilities module allows for the description of abilities and their
+dependencies across Archivematica components (i.e., dashboard, MCPServer, and
+MCPClient).
+"""
```

Comment:
This is the `abilities` module that used to be part of archivematicaCommon but now moved in MCPServer because we're not sharing it anymore since workflow responsibilities are now carried just by MCPServer and exposed via the API!